server/sonar-server/src/main/java/org/sonar/server/permission/ws/template/SetDefaultTemplateAction.java

```diff
@@ -90,7 +89,7 @@ private void doHandle(SetDefaultTemplateWsRequest request) {
       PermissionTemplateDto template = findTemplate(dbSession, request);
       checkGlobalAdmin(userSession, template.getOrganizationUuid());
       validateQualifier(qualifier, resourceTypes);
```

Comment:
when governance is not installed, the view qualifier must be rejected, so, we still need resourceTypes